lib/index.js

```diff
@@ -36,19 +36,39 @@ function createDiaryURL(date, category) {
 }
 
 function fetchBreakfast(date) {
-  $.getJSON(createDiaryURL(date, 'breakfast'), createFoodHandler('#breakfast'));
+  $.getJSON(createDiaryURL(date, 'breakfast'), (data) => {
+    createFoodHandler('#breakfast')(data)
+    totalMealCalories('#breakfast');
+    dailyCalories();
+    remainingDailyCalories();
+  })
 }
 
 function fetchLunch(date) {
-  $.getJSON(createDiaryURL(date, 'lunch'), createFoodHandler('#lunch'));
+  $.getJSON(createDiaryURL(date, 'lunch'), (data) => {
+    createFoodHandler('#lunch')(data)
+    totalMealCalories('#lunch');
+    dailyCalories();
+    remainingDailyCalories();
+  })
 }
 
 function fetchDinner(date) {
-  $.getJSON(createDiaryURL(date, 'dinner'), createFoodHandler('#dinner'));
+  $.getJSON(createDiaryURL(date, 'dinner'), (data) => {
+    createFoodHandler('#dinner')(data)
+    totalMealCalories('#dinner');
+    dailyCalories();
+    remainingDailyCalories();
+  })
 }
 
 function fetchSnacks(date) {
-  $.getJSON(createDiaryURL(date, 'snacks'), createFoodHandler('#snacks'));
+  $.getJSON(createDiaryURL(date, 'snacks'), (data) => {
```

Comment:
These functions aren't feeling super DRY, do you think we could refactor into a method that passes the meal name through as an argument?